std/assembly/builtins.ts

```diff
@@ -1001,13 +1001,25 @@ export namespace v128 {
   @builtin
   export declare function shuffle<T>(a: v128, b: v128, ...lanes: u8[]): v128;
 
+  // @ts-ignore: decorator
+  @builtin
+  export declare function swizzle(a: v128, s: v128): v128;
+
   // @ts-ignore: decorator
   @unsafe @builtin
-  export declare function load(offset: usize, immOffset?: usize, immAlign?: usize): v128;
+  export declare function load(ptr: usize, immOffset?: usize, immAlign?: usize): v128;
 
   // @ts-ignore: decorator
   @unsafe @builtin
-  export declare function store(offset: usize, value: v128, immOffset?: usize, immAlign?: usize): void;
+  export declare function load_splat<T>(ptr: usize, immOffset?: usize, immAlign?: usize): v128;
+
+  // @ts-ignore: decorator
+  @unsafe @builtin
+  export declare function load_ext<TFrom>(ptr: usize, immOffset?: usize, immAlign?: usize): v128;
```

Comment:
Why `TFrom` instead `T`?

Reply:
The instruction extends lanes of one type to the respective next larger type (e.g. 8 lanes `u8` to 8 lanes `u16`), so I named it `TFrom` to give a hint that the smaller type is the type argument and the larger type is implicit.

Reply:
Got it